website/docs/d/express_route_circuit_connections.html.markdown

```diff
@@ -0,0 +1,47 @@
+---
+subcategory: "Network"
+layout: "azurerm"
+page_title: "Azure Resource Manager: Data Source: azurerm_express_route_circuit_connection"
+description: |-
+  Gets information about an existing network ExpressRouteCircuitConnection.
+---
+
+# Data Source: azurerm_express_route_circuit_connection
+
+Use this data source to access information about an existing network ExpressRouteCircuitConnection.
```

Comment:
can we add a proper description here

Reply:
I think we don't need data source for this RP. So I removed this file.